_includes/v3.0/release-notes/v3.0.5-release-notes.md

```diff
@@ -0,0 +1,38 @@
+16 April 2018
+
+> **Important**: [Review the documentation](https://docs.projectcalico.org/v3.0/getting-started/kubernetes/upgrade/) carefully before attempting an upgrade.
+> You must upgrade to Calico v2.6.5 or later before you can upgrade to Calico v3.0.4.
+{: .alert .alert-danger}
+
+#### Bug Fixes
+
+- Add support for configuring container IP forwarding via the CNI configuration file. [cni-plugin #508](https://github.com/projectcalico/cni-plugin/pull/508) (@caseydavenport)
+- When auto-detecting a node name, Calico will convert the hostname to lowercase. [calico #1880](https://github.com/projectcalico/calico/pull/1880) (@robbrockbank)
+- Fix bug [typha #118](https://github.com/projectcalico/typha/issues/118) where calico/node and Typha race to create the cluster information resource, causing Typha to crash. [libcalico-go #845](https://github.com/projectcalico/libcalico-go/pull/845) (@robbrockbank)
+- Improve confd behavior when using the Kubernetes API datastore and node-to-node mesh disabled. [libcalico-go #838](https://github.com/projectcalico/libcalico-go/pull/838) (@caseydavenport)
```

Comment:
This is not quite a sentence.....perhaps consider something more like, "Improve confd behavior when using the Kubernetes API datastore with node-to-node mesh disabled."